apps/website/src/pages/docs/transferlist.mdx

```diff
@@ -12,22 +12,12 @@ import LiveExample from '~/components/LiveExample.astro';
 import RelatedComponents from '~/components/RelatedComponents.astro';
 import * as AllExamples from 'examples';
 
-<p>{frontmatter.description}</p>
+<p>{frontmatter.description} The labels should always be provided above the TransferList.</p>
```

Comment:
nit: You can add that sentence below the `<p>` tag and before the `<LiveExample>`.